playground/nextjs-app-router/tailwind.config.ts

```diff
@@ -6,6 +6,7 @@ const config: Config = {
     './components/**/*.{js,ts,jsx,tsx,mdx}',
     './app/**/*.{js,ts,jsx,tsx,mdx}',
   ],
+  safelist: ['dark'],
```

Comment:
Need this or your local preference will apply to the component.

https://github.com/coinbase/onchainkit/pull/1017/files#diff-3c4635766577fdb96b55f38b217b1e9577b4717264f947e430e86fb8c1dc7182R23